src/BLSApkRegistry.sol

```diff
@@ -148,6 +200,34 @@ contract BLSApkRegistry is BLSApkRegistryStorage {
     /*******************************************************************************
                             VIEW FUNCTIONS
     *******************************************************************************/
+    /**
+     * @notice Returns the pubkey and pubkey hash of an operator
+     * @dev Reverts if the operator has not registered a valid pubkey
+     */
+    function getRegisteredPubkey(address operator) public view returns (BN254.G1Point memory, bytes32) {
+        BN254.G1Point memory pubkey = operatorToPubkey[operator];
+        bytes32 pubkeyHash = operatorToPubkeyHash[operator];
+
+        require(
+            pubkeyHash != bytes32(0),
+            "BLSApkRegistry.getRegisteredPubkey: operator is not registered"
+        );
+        
+        return (pubkey, pubkeyHash);
+    }
+
+    /**
+     * @notice Returns the message hash that an operator must sign to register their BLS public key.
+     * @param operator is the address of the operator registering their BLS public key
+     */
+    function getMessageHash(address operator) public view returns (BN254.G1Point memory) {
+        return BN254.hashToG1(keccak256(abi.encodePacked(
+            operator, 
+            address(this),
+            block.chainid, 
+            "EigenLayer_BN254_Pubkey_Registration"
```

Comment:
Should probably change the message hash "EigenLayer_BN254_Pubkey_Registration" to something that's more AVS specific, or just remove entirely since there won't be a singleton compendium contract anymore

Reply:
hmm, perhaps we can just delete the `EigenLayer_` part?
it's overkill to have this at all, but it's another input to help protect from any kind of replay issues, so...I feel like it's not a bad idea to have _some_ string here at least.

Reply:
realized I should probably just convert this to use EIP-712 instead; will add that to my TODOs

Reply:
addressed as part of https://github.com/Layr-Labs/eigenlayer-middleware/pull/102 -- I think these changes are quite appropriate